README.md

```diff
@@ -1,102 +1,191 @@
+* Rolling upgrades may be possible, but they will be subject to any
+  limitations imposed by new features.
 
 ## <a name="building"></a>Building this Release from Source
 
-Directions to build Apache Geode from source are in the source distribution, file `BUILDING.md`.
+See [BUILDING.md](https://github.com/apache/geode/blob/develop/BUILDING.md) for
+instructions on how to build the project.
 
 ## <a name="started"></a>Geode in 5 minutes
 
-With a JDK version 1.8 or a more recent version installed,
+Geode requires installation of JDK version 1.8.  After installing Apache Geode,
 start a locator and server:
 
     $ gfsh
-    gfsh> start locator --name=locator
-    gfsh> start server --name=server
+    gfsh> start locator
+    gfsh> start server
 
 Create a region:
 
-    gfsh> create region --name=region --type=REPLICATE
+    gfsh> create region --name=hello --type=REPLICATE
+
+Write a client application (this example uses a [Gradle](https://gradle.org)
+build script):
 
-Write a client application:
+_build.gradle_
+
+    apply plugin: 'java'
+    apply plugin: 'application'
+
+    mainClassName = 'HelloWorld'
+
+    repositories { mavenCentral() }
+    dependencies {
+      compile 'org.apache.geode:geode-core:1.1.0'
+      runtime 'org.slf4j:slf4j-log4j12:1.7.24'
+    }
 
-_HelloWorld.java_
+_src/main/java/HelloWorld.java_
 
     import java.util.Map;
     import org.apache.geode.cache.Region;
     import org.apache.geode.cache.client.*;
-    
+
     public class HelloWorld {
       public static void main(String[] args) throws Exception {
         ClientCache cache = new ClientCacheFactory()
           .addPoolLocator("localhost", 10334)
           .create();
         Region<String, String> region = cache
           .<String, String>createClientRegionFactory(ClientRegionShortcut.CACHING_PROXY)
-          .create("region");
-    
+          .create("hello");
+
         region.put("1", "Hello");
         region.put("2", "World");
-    
+
         for (Map.Entry<String, String>  entry : region.entrySet()) {
           System.out.format("key = %s, value = %s\n", entry.getKey(), entry.getValue());
         }
         cache.close();
       }
     }
 
-Compile and run `HelloWorld.java`.  The classpath should include `geode-dependencies.jar`.
+Build and run the `HelloWorld` example:
+
+    $ gradle run
 
-    javac -cp /some/path/geode/geode-assembly/build/install/apache-geode/lib/geode-dependencies.jar HelloWorld.java
-    java -cp .:/some/path/geode/geode-assembly/build/install/apache-geode/lib/geode-dependencies.jar HelloWorld
+The application will connect to the running cluster, create a local cache, put
+some data in the cache, and print the cached data to the console:
+
+    key = 1, value = Hello
+    key = 2, value = World
+
+For more information see the [Geode
+Examples](https://github.com/apache/geode-examples) repository or the
+[documentation](http://geode.apache.org/docs/).
```

Comment:
Please include a 
  gfsh shutdown --include-locators=true
command.  Otherwise new users won't realize that they still have a locator and server running.  Makes it especially confusing if they start further locators and servers, since leaving out the --name option (in the gfsh start commands) won't cause a complaint.